src/DaysAccordion.tsx

```diff
@@ -0,0 +1,242 @@
+                  <TableCell>{exercise.name}</TableCell>
+                  <TableCell align="right">{exercise.sets}</TableCell>
+                  <TableCell align="right">{exercise.reps}</TableCell>
+                  <TableCell align="right">{exercise.weight}</TableCell>
+                </TableRow>
+              ))}
+            </TableBody>
+          </Table>
+        </AccordionDetails>
+      </Accordion>
+      <Accordion defaultExpanded>
+        <AccordionSummary
+          expandIcon={<ExpandMoreIcon />}
+          aria-controls="panel2bh-content"
+          id="panel2bh-header"
+        >
+          <Typography>Tuesday</Typography>
+        </AccordionSummary>
+        <AccordionDetails>
+          <Table>
+            <TableHead>
+              <TableRow>
+                <TableCell>Name</TableCell>
+                <TableCell align="right">Sets</TableCell>
+                <TableCell align="right">Reps</TableCell>
+                <TableCell align="right">Weight</TableCell>
+              </TableRow>
+            </TableHead>
+            <TableBody>
+              {exercises.tuesday.map((exercise: Exercise) => (
+                <TableRow key={exercise.id}>
+                  <TableCell>{exercise.name}</TableCell>
+                  <TableCell align="right">{exercise.sets}</TableCell>
+                  <TableCell align="right">{exercise.reps}</TableCell>
+                  <TableCell align="right">{exercise.weight}</TableCell>
+                </TableRow>
+              ))}
+            </TableBody>
+          </Table>
+        </AccordionDetails>
+      </Accordion>
+      <Accordion defaultExpanded>
+        <AccordionSummary
+          expandIcon={<ExpandMoreIcon />}
+          aria-controls="panel3bh-content"
+          id="panel3bh-header"
+        >
+          <Typography>Wednesday</Typography>
+        </AccordionSummary>
+        <AccordionDetails>
+          <Table>
+            <TableHead>
+              <TableRow>
+                <TableCell>Name</TableCell>
+                <TableCell align="right">Sets</TableCell>
+                <TableCell align="right">Reps</TableCell>
+                <TableCell align="right">Weight</TableCell>
+              </TableRow>
+            </TableHead>
+            <TableBody>
+              {exercises.wednesday.map((exercise: Exercise) => (
+                <TableRow key={exercise.id}>
+                  <TableCell>{exercise.name}</TableCell>
+                  <TableCell align="right">{exercise.sets}</TableCell>
+                  <TableCell align="right">{exercise.reps}</TableCell>
+                  <TableCell align="right">{exercise.weight}</TableCell>
+                </TableRow>
+              ))}
+            </TableBody>
+          </Table>
+        </AccordionDetails>
+      </Accordion>
+      <Accordion defaultExpanded>
+        <AccordionSummary
+          expandIcon={<ExpandMoreIcon />}
+          aria-controls="panel4bh-content"
+          id="panel4bh-header"
```

Comment:
Don't need these ID and aria-controls properties. Same for everywhere else they're used in this file.